services/ui/ws/window_tree.cc

```diff
@@ -900,6 +900,13 @@ void WindowTree::SendToPointerWatcher(const ui::Event& event,
                                    display_id);
 }
 
+WindowManagerState* WindowTree::AppendExternalModeWindowManagerState(
```

Comment:
I think we could just return void here.

I know that on the caller site needs a pointer of WindowManagerState, so I would rather 

create unique_ptr<WindowManagerState> in display::InitDisplayRoot
save a raw pointer temporary in a variable
call AppendExternalModeWindowManagerState (maybe other methods use "Add" suffix rather than "Append"?)


do